patches/tModLoader/Terraria/Wiring.cs.patch

```diff
@@ -50,6 +50,31 @@
  		private static int CurrentUser = 254;
  
  		public static void SetCurrentUser(int plr = -1) {
+@@ -226,21 +_,21 @@
+ 
+ 		public static bool Actuate(int i, int j) {
+ 			Tile tile = Main.tile[i, j];
+-			if (!tile.actuator())
++			if (!tile.actuator() || !TileLoader.CanActuate(tile))
```

Comment:
Shouldn't CanActuate override tile.actuator? I thought the intention was to provide the possibly for actuation without vanilla logic.